### ISSUE TITLE
Add fields in BloodRequestDTO

### ISSUE THREAD

**Issue by Code-4ge** [taha-yasin/blood-donor-backend#11]
Add some fields in BloodRequestDTO file like: patient age, gender, unitOfBloodRequired.

**Comment by taha-yasin** [comment]
> Add some fields in BloodRequestDTO file like: patient age, gender, unitOfBloodRequired.

JSON format for Request:
```
{
        patientName:"",
        requiredBloodGroup:"",
        whatsapp:"",
        gender:"",
        age:"",
        unitRequired:"",
        streetAddress:"",
        city:"",
        govermentId:"",
        donors:[],
}
```
Is this the final format required ??

**Comment by Code-4ge** [comment]
> > Add some fields in BloodRequestDTO file like: patient age, gender, unitOfBloodRequired.
> 
> JSON format for Request:
> 
> ```
> {
>         patientName:"",
>         requiredBloodGroup:"",
>         whatsapp:"",
>         gender:"",
>         age:"",
>         unitRequired:"",
>         streetAddress:"",
>         city:"",
>         govermentId:"",
>         donors:[],
> }
> ```
> 
> Is this the final format required ??

**Yes**,
If we are not considering doctors prescription.

**Comment by taha-yasin** [comment]
fixed in pull request #14
